perf/istio/setup.sh

```diff
@@ -106,6 +106,13 @@ function setup_admin_binding() {
     --user=$(gcloud config get-value core/account) || true
 }
 
+function inject_workload() {
+  local deployfile="${1:?"specify the workload deployment file"}"
+
+  istioctl kube-inject -f "${deployfile}" -o temp-workload-injected.yaml
```

Comment:
Not sure why this is here. We use side car injector.

Reply:
Moved to sds-tests/setup_test.sh.